packages/web-components/src/components/cbp-usa-banner/cbp-usa-banner.scss

```diff
@@ -0,0 +1,66 @@
+/**
+ * @Prop   --cbp-usa-banner-background-color: var(--cbp-color-gray-cool-90);
+ * @Prop   --cbp-usa-banner-color: var(--cbp-color-text-lightest);
+ */
+ 
+ :root{
+  --cbp-usa-banner-background-color: var(--cbp-color-gray-cool-90);
+  --cbp-usa-banner-color: var(--cbp-color-text-lightest);
+}
+
+cbp-usa-banner {
+  background-color: var(--cbp-usa-banner-background-color);
+  color: var(--cbp-usa-banner-color);
+  display: inline-block;
+  width: 100%;
+  padding-block: var(--cbp-space-4x);
+  padding-inline: var(--cbp-space-8x);
+
+  --cbp-usa-banner-icon-rotate: rotate(90deg);
+
+  &:not([open]) .cbp-usa-banner-content {
```

Comment:
this is a little convoluted. I would just set it to display none and then display block with the [open] selector below.

Reply:
removed the :not(open) here